### PR TITLE
added cathy command

### PR DESCRIPTION
**Pull request by thomasgilgenast**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -517,6 +517,10 @@ function eve(bot, message) {
     })
 }
 
+function cathy(bot, message) {
+    bot.reply(message, 'Secretary of State, Catharine Durant');
```
Comment (michaeltritz) [inline app.js]: Catherine*

File: app.js [hunk 2]
```diff
@@ -541,6 +545,7 @@ controller.hears('^!wowstats (.*)$', defaultContexts, wowstats);
 controller.hears('^!twitch', defaultContexts, twitch);
 controller.hears('^!antitritz$', defaultContexts, antiTritz);
```
Comment (michaeltritz) [inline app.js]: delete this

File: app.js (hunk 1 above)
Comment (thomasgilgenast) [inline app.js]: good catch

File: app.js (hunk 2 above)
Comment (thomasgilgenast) [inline app.js]: dismissing this request as irrelevant to the scope of the proposed feature

**Pull request merged**